From 89512ec5c49de8d29e25d2758f7065ded09a1bd9 Mon Sep 17 00:00:00 2001
From: Dan Williams <dcbw@redhat.com>
Date: Fri, 19 Jul 2013 18:32:44 +0000
Subject: build: allow dhcpcd versions higher than 5

No reason to exclude them, just that configure.ac hasn't tracked new
versions of dhcpcd.
---
diff --git a/configure.ac b/configure.ac
index d28e4bf..6b70bbd 100644
--- a/configure.ac
+++ b/configure.ac
@@ -506,8 +506,8 @@ fi
 if test "$with_dhcpcd" = "yes"; then
 	AC_PATH_PROGS(with_dhcpcd, dhcpcd, no, /sbin:/usr/sbin:/usr/local/sbin)
 	if test "$with_dhcpcd" != "no"; then
-		if ! $with_dhcpcd --version 2>&1 | grep -q "^dhcpcd [[45]]\."; then
-			AC_MSG_WARN([Cannot use dhcpcd, version 4.x or 5.x is required])
+		if ! $with_dhcpcd --version 2>&1 | grep -q "^dhcpcd [[456789]]\."; then
+			AC_MSG_WARN([Cannot use dhcpcd, version 4.x or higher is required])
 			with_dhcpcd=no
 		fi
 	fi
--
cgit v0.9.0.2-2-gbebe
